core/schema_utils.py

```diff
@@ -681,3 +683,51 @@ def is_valid_username_string(obj: str) -> bool:
             return True
         except utils.ValidationError:
             return False
+
+    @staticmethod
+    def is_valid_story_title(obj: str) -> bool:
```

Comment:
@vojtechjelinek We should really define the scope of schema_utils i.e, what can be defined in this file? Do you think we should keep `story` keyword in the schema_utils?

Reply:
I don't have strict opinions about this. I'm fine both defining it in schema utils but also I'm fine not allowing it and passing it there.

Reply:
I removed the `story` keyword and validation out of `schema_utils` as it is causing an issue with cyclic imports, also the reason some tests are failing.